azure-pipelines.yml

```diff
@@ -0,0 +1,22 @@
+resources:
+- repo: self
+
+pool:
+  vmimage: 'ubuntu-16.04'
+
+trigger:
+- master
+
+steps:
+- script: |
+    docker run --rm -v '$(Build.SourcesDirectory):/mnt' -t koalaman/shellcheck aks/entrypoint.sh
```

Comment:
This script doesn't seem scalable. If a new action is introduced later, the action's author would have to contribute to this file as well.

You could get all the `.sh` files, by running this command: `find . -name "*.sh"`, and iterate over the results.